wagtail/snippets/views/snippets.py

```diff
@@ -1071,6 +1109,18 @@ def get_menu_order(self):
     def get_menu_item_is_registered(self):
         return self.menu_item_is_registered
 
+    @cached_property
+    def breadcrumb_items(self):
```

Comment:
This allows us to share the first two items for all views instead of duplicating them.